4week/src/hooks/useDebounce.ts

```
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+
+/**
+ * @param value 디바운싱할 대상 값
+ * @param delay 디바운싱 지연 시간 (ms)
+ * @returns 디바운싱이 적용된 값
+ */
+export const useDebounce = <T>(value: T, delay: number): T => {
+  // 디바운싱된 값을 저장할 state
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)